packages/manager/modules/pci/src/projects/project/notebooks/add/add.controller.js

```diff
@@ -24,4 +103,133 @@ export default class {
   refreshMessages() {
     this.messages = this.messageHandler.getMessages();
   }
+
+  getConfigHeader(currentStep) {
+    if (currentStep > 0) {
+      return this.$translate.instant(
+        'pci_notebook_add_configuration_title_selected',
+        {
+          editor: this.notebookModel.selected.editor.name,
+        },
+      );
+    }
+
+    return this.$translate.instant('pci_notebook_add_configuration_title');
+  }
+
+  getFrameworkHeader(currentStep) {
+    if (currentStep > 1) {
+      return this.$translate.instant(
+        'pci_notebook_add_framework_title_selected',
+        {
+          framework: this.notebookModel.selected.framework.model.name,
+          version: this.notebookModel.selected.framework.version.version,
+        },
+      );
+    }
+
+    return this.$translate.instant('pci_notebook_add_framework_title');
+  }
+
+  getPrivacyHeader(currentStep) {
+    if (currentStep > 2) {
+      const privacy = this.$translate.instant(
+        `pci_notebook_add_privacy_settings_item_selected_${this.notebookModel.selected.privacy}`,
+      );
+      return this.$translate.instant(
+        `pci_notebook_add_privacy_settings_title_selected`,
+        {
+          privacy,
+        },
+      );
+    }
+
+    return this.$translate.instant('pci_notebook_add_privacy_settings_title');
+  }
+
+  getRegionHeader(currentStep) {
+    if (currentStep > 3) {
```

Comment:
So if we change the steps order we need to update all the functions, I wonder if it should be better to condition the display in the template in that case

Reply:
Yes, i know, it same even if we implement it into html.
I preferred to implement it into JS side because it will more readable than to implement a series of ternary operator inside a html.